--- Makefile.in~	Tue Jun 11 01:14:53 2002
+++ Makefile.in	Thu Dec  5 15:37:14 2002
@@ -248,22 +248,22 @@
 #========================================================================
 
 install-libraries: libraries
-	$(mkinstalldirs) $(pkglibdir)
-	@echo "Installing modules in $(pkglibdir)"
+	$(mkinstalldirs) $(DESTDIR)$(pkglibdir)
+	@echo "Installing modules in $(DESTDIR)$(pkglibdir)"
 	@for i in $(MODULES) ; do \
 	    if test -d $(srcdir)/modules/$$i; then  \
-		echo "Installing $$i in $(pkglibdir)/$$i" ; \
-		$(mkinstalldirs) $(pkglibdir)/$$i ; \
+		echo "Installing $$i in $(DESTDIR)$(pkglibdir)/$$i" ; \
+		$(mkinstalldirs) $(DESTDIR)$(pkglibdir)/$$i ; \
 		for j in `ls $(srcdir)/modules/$$i/*.tcl 2>/dev/null` ; do \
-		    $(INSTALL_DATA) $$j $(pkglibdir)/$$i ; \
+		    $(INSTALL_DATA) $$j $(DESTDIR)$(pkglibdir)/$$i ; \
 		done; \
 		if test -f $(srcdir)/modules/$$i/tclIndex ; then \
-		    $(INSTALL_DATA) $(srcdir)/modules/$$i/tclIndex $(pkglibdir)/$$i ; \
+		    $(INSTALL_DATA) $(srcdir)/modules/$$i/tclIndex $(DESTDIR)$(pkglibdir)/$$i ; \
 		fi \
 	    fi; \
 	done;
 	$(MAKE) gen-main-index "PKGDIR=$(srcdir)"
-	cp $(srcdir)/pkgIndex.tcl $(pkglibdir)
+	cp $(srcdir)/pkgIndex.tcl $(DESTDIR)$(pkglibdir)
 
 #========================================================================
 # Install documentation.  Unix manpages should go in the $(mandir)
@@ -273,12 +273,12 @@
 install-doc: doc
 	@echo Installing documentation for $(PACKAGE)
 	@if [ `echo 'puts $$tcl_platform(platform) ; exit' | $(TCLSH_PROG)` = unix ] ; then \
-	    $(mkinstalldirs) $(mandir)/mann ; \
-	    for f in doc/nroff/*.n ; do cp -f $$f $(mandir)/mann ; done ; \
-	    $(INSTALL_DATA) $(srcdir)/man.macros $(mandir) ; \
+	    $(mkinstalldirs) $(DESTDIR)$(mandir)/mann ; \
+	    for f in doc/nroff/*.n ; do cp -f $$f $(DESTDIR)$(mandir)/mann ; done ; \
+	    $(INSTALL_DATA) $(srcdir)/man.macros $(DESTDIR)$(mandir) ; \
 	fi
-	$(mkinstalldirs)   $(pkglibdir)/htmldoc
-	cp doc/html/*.html $(pkglibdir)/htmldoc
+	$(mkinstalldirs)   $(DESTDIR)$(pkglibdir)/htmldoc
+	cp doc/html/*.html $(DESTDIR)$(pkglibdir)/htmldoc
 
 test:
 	$(TCLSH_PROG) `$(CYGPATH) $(srcdir)/all.tcl` -modules "$(MODULES)"
