--- tcllib-1.15/installer.tcl.orig	2013-01-23 05:25:49.000000000 +0100
+++ tcllib-1.15/installer.tcl	2014-01-05 07:29:01.619314580 +0100
@@ -218,7 +218,7 @@
 
 	if {[file exists $aexe.man]} {
 	    if {$config(doc,nroff)} {
-		_manfile $aexe.man nroff n $config(doc,nroff,path)
+		_manfile $aexe.man nroff 1 $config(doc,nroff,path)
 	    }
 	    if {$config(doc,html)} {
 		_manfile $aexe.man html html $config(doc,html,path)
